ghost/core/core/server/web/admin/app.js

```diff
@@ -52,9 +52,7 @@ module.exports = function setupAdminApp() {
         } catch (err) {
             next(err);
         }
-    }, serveStatic(
-        path.join(config.getContentPath('public'), 'admin-auth')
-    ));
+    }, createServeAuthFrameFileMw(config, urlUtils));
```

Comment:
Had to switch to using a custom middleware as `express/serve-static` does not allow us to hook into the response body to perform the necessary placeholder replacements